src/config/passport.js

```diff
@@ -1,28 +1,30 @@
-const passport = require("passport");
-const LocalStrategy = require("passport-local").Strategy;
-const mongoose = require("mongoose");
-const User = mongoose.model("User");
 import dotenv from 'dotenv';
+
+const passport = require('passport');
+const LocalStrategy = require('passport-local').Strategy;
+const mongoose = require('mongoose');
```

Comment:
I thought we have decided to remove redundant code like mongoose?

Reply:
Yes. @chis0m is working with the passport.js file to create the social provider authentication is I left the boiler plate for him

Reply:
Okay